py3status/modules/sysdata.py

```diff
@@ -138,55 +144,90 @@ class Py3status:
     cache_timeout = 10
     format = "[\?color=max_cpu_mem CPU: {cpu_usage}%, " \
         "Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]"
-    padding = 0
-    precision = 2
-    thresholds = None
+    format_flat = "[\?min_length=0 {value:.2f}]"
```

Comment:
Can't these just be `{value:.2f}`?

I'm not sure about the name `format_flat` it is not very expressive from what I can tell